internal/configs/version1/nginx-plus.ingress.tmpl

```diff
@@ -136,15 +141,19 @@ server {
 
 	{{- range $location := $server.JWTRedirectLocations}}
 	location {{$location.Name}} {
+		set $service "nil";
```

Comment:
Looks like in some places such as here, the default value is "nil", whereas in other places it's "". Is there a need to have "nil" instead of ""?

Reply:
you need a value in between the `""` otherwise nginx conf will generate that like as `set $service ;` and cause errors and not load/reload properly, `nil` was the first one that came to mind. But it turns out that its not needed there or in some other places anyway so I removed them

Reply:
Thanks for explaining @LorcanMcVeigh !